### DISCUSSION TITLE
802.11 Rate Control - Showcase

### DISCUSSION THREAD

**Discussion by rhornig** [inet-framework/inet#591] [Showcases]
Discussion about the 802.11 Rate Control showcase

**Comment by mhdsaki** [comment]
Hi,
The throughput vs time never change when I change some parameters such as the opMode from "g" to "ac" or initial rate, etc. It seems that no parameters have any effect on the maximum throughput!
Could anyone please share any helpful idea?
Thanks

**Comment by levy** [comment]
In this example, the data rate is controlled by the 802.11 MAC, so setting the initial rate will not have too much effect. Even if you set to "ac" the selected data rate may still be the same due to channel conditions. I haven't checked this particular case though.

**Comment by mhdsaki** [comment]
Thanks. But is there totally any way to have the results with higher data rates?

Kind regards
Mahdi Saki

On 26 Oct 2018, at 18:51, Levente Mészáros <notifications@github.com<mailto:notifications@github.com>> wrote:


In this example, the data rate is controlled by the 802.11 MAC, so setting the initial rate will not have too much effect. Even if you set to "ac" the selected data rate may still be the same due to channel conditions. I haven't checked this particular case though.

**Comment by alya831** [comment]
Hi,
I was run the AARF Rate Control, and the results were same  as shown.
For 'AARF Rate Control', the throughput and bit rate recorded and plotted very well. But, the bit rate for 'No Rate Control' was not recorded. 
Could anyone please share any idea?
Thanks,
